### ISSUE TITLE
WasapiOut.Init for changing source file not working

### ISSUE THREAD

**Issue by robymaz** [naudio/NAudio#506]
Hi!

we are facing some problems while changing source file to be played with a WasapiOut object. We are using Naudio libraries for a windows application: what we do is calling the WasapiOut.Init method each time we need to change the source file without creating a new WasapiOut object. 
This way has always worked but it seems that for some days now there have been problems that are most likely due to windows or drivers updates.
We have tried this solution: disposing the WasapiOut object and create a new one each time we need to change the wav file to be played, and calling Init on the new object only once. It seems it is working. Is it the correct solution? Do you see any possible drawbacks of this solution? Is there a better way to solve the issue?

Thank you very much in advance
Roberta

**Comment by markheath** [comment]
Yes, you are only supposed to call `Init` once on an instance of `WasapiOut`. Perhaps in future we could make it able to support multiple initializations
